main.go

```diff
@@ -668,6 +682,11 @@ func clientBackendDialer(cert certloader.Certificate, network, address, host str
 // target. The input can be or the form "HOST:PORT" for TCP or "unix:PATH"
 // for a UNIX socket.
 func parseUnixOrTCPAddress(input string) (network, address, host string, err error) {
+	if strings.HasPrefix(input, "launchd") {
```

Comment:
Test for equality instead? Otherwise `--listen launchdfoobar` would work which seems counter-intuitive.